File: ABC180/B.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int N;
  cin >> N;
  
  vector<int> x(N);
  for(int i=0; i<N;i++){
    cin >> x.at(i);
  }
  
  long long man=0;
  for(int i=0; i<N;i++){
    man+=abs(x[i]);
  }
  cout << man << endl;
  
  long double eu=0;
  for(int i=0; i<N;i++){
    eu+=pow(x[i],2);
  }
  cout << fixed << setprecision(15) << sqrt(eu) << endl;
  
  int che=0;
  for(int i=0; i<N;i++){
    che=max(che,abs(x[i]));
  }
  cout << che << endl;
}

File: ABC180/B.py
import math
import sys
sys.setrecursionlimit(10**6)

N=int(input())
x=list(map(int, input().split()))

man=0
for i in range(N):
  man+=abs(x[i])
print(man)

eu=0
for i in range(N):
  eu+=pow(x[i],2)
print(math.sqrt(eu))

print(max([abs(i) for i in x]))
